qa/L0_io/test.sh

```diff
@@ -78,7 +78,36 @@ for trial in graphdef savedmodel netdef onnx libtorch plan ; do
     fi
 done
 
-# custom model needs to be obtained elsewhere
+# Prepare string models with basic config
+for trial in graphdef savedmodel onnx ; do
+    full=${trial}_object_object_object
+    mkdir -p $MODELSDIR/${full}/1 && \
+        cp -r $DATADIR/${full}/1/* $MODELSDIR/${full}/1/. && \
+        cp $DATADIR/${full}/config.pbtxt $MODELSDIR/${full}/. && \
+        (cd $MODELSDIR/${full} && \
+                sed -i "s/label_filename:.*//" config.pbtxt && \
+                echo "instance_group [{ kind: KIND_CPU }]" >> config.pbtxt)
+
+    # ensemble version of the model.
+    mkdir -p $MODELSDIR/fan_${full}/1 && \
+    cp $ENSEMBLEDIR/fan_${full}/config.pbtxt $MODELSDIR/fan_${full}/. && \
+        (cd $MODELSDIR/fan_${full} && \
+                sed -i "s/label_filename:.*//" config.pbtxt)
+
+    if [ "$trial" == "libtorch" ]; then
```

Comment:
Are you planning on adding libtorch string model?

Reply:
Oops, removing this, libtorch does not support string